common/utils_unittest.cpp

```diff
@@ -126,9 +126,9 @@ UTILS_TEST(guessResolutionOverflow) {
             << "x"
             << std::numeric_limits<long>::max();
 
-    EXPECT_FALSE(guessResolution(sstream.str().c_str(), w, h));
-    EXPECT_EQ(w, 0);
-    EXPECT_EQ(h, 0);
+    EXPECT_TRUE(guessResolution(sstream.str().c_str(), w, h));
```

Comment:
could we make w, h = 0 in this case? Since we will return 0 for "w3840h2160",  better return 0 if we can't get enough room for long.